src/dotnet/APIView/APIViewWeb/Helpers/APIHelpers.cs

```diff
@@ -1,15 +1,54 @@
 using System;
-using System.Collections.Generic;
-using System.Linq;
 using System.Text.Json.Serialization;
 using System.Text.Json;
 using System.Threading.Tasks;
 using Microsoft.AspNetCore.Mvc;
-using Microsoft.Azure.Cosmos.Linq;
-using Microsoft.TeamFoundation.SourceControl.WebApi;
+using System.Collections.Generic;
 
 namespace APIViewWeb.Helpers
 { 
+    public class PageParams
+    {
+        private const int MaxPageSize = 50;
+        public int NoOfItemsRead { get; set; } = 0;
+        private int _pageSize = 5;
+
+        public int PageSize { 
+            get => _pageSize; 
+            set => _pageSize = (value > MaxPageSize) ? MaxPageSize : value;
+        }
+    }
+
+    public class ReviewFilterAndSortParams
+    {
+        public string Name { get; set; }
```

Comment:
Is it author name or package name?

Reply:
PackageName. Author filter can be added to Details field